packages/shared/components/dashboard/dashboard-portfolio-events/dashboard-portfolio-events.tsx

```diff
@@ -16,7 +16,15 @@ import withLoader, { WithLoaderProps } from "shared/decorators/with-loader";
 
 import DashboardPortfolioEventsListLoader from "./dashboard-portfolio-event-loader/dashboard-portfolio-event-list-loader";
 
-const DASHBOARD_EVENTS_STYLE = { height: "100%", minHeight: "450px" };
+const DASHBOARD_EVENTS_STYLE = {
```

Comment:
Could you investigate what is wrong if production works well?

Reply:
![image](https://user-images.githubusercontent.com/17555617/58542961-2940b800-8207-11e9-8781-6ff52f3f23ce.png)
There is overflow on the tablet screen size after your changes